packages/next/client/components/redirect.ts

```diff
@@ -0,0 +1,14 @@
+export const REDIRECT_ERROR_CODE = 'NEXT_REDIRECT'
```

Comment:
Maybe we should throw a `Symbol` instead of a `string`? `Symbol('Next.js Redirect')`.

Reply:
Since this file will be bundled in more than one environment it can 2 different instances at times hence why it's a string